examples/server/example-server-node.ts

```diff
@@ -1,6 +1,6 @@
-import * as net from 'net'
-import * as stream from 'stream'
-import * as WebSocket from 'ws'
+import net from 'net'
+import stream from 'stream'
```

Comment:
Are you sure these had to be changed? I don't think this change is correct. `net` and `stream` are whole-module imports, not default exports

Reply:
No, I'm not sure. I `codedmod`ded all of these. I'll go back and fix each one individually instead.